src/main/kotlin/Interfaces.kt

```diff
@@ -19,14 +19,15 @@ interface Drivable {
 interface TimeTravelable {
     var charged: Boolean
     fun chargeFluxCapacitor()
+    fun inspectFluxCapacitor()
```

Comment:
I added this to demonstrate an example in the implementation below. Also because it's fun. :)